javascript/javascript1/week2/homework-week2.js

```diff
@@ -0,0 +1,126 @@
+//Flight booking fullname function
+
+function getFullname (firstname, surname){
+    return firstname + " " + surname;
+}
+var fullname1 = getFullname("Benjamin", "Hughes");
+var fullname2 = getFullname("Atefeh", "Behroozi");
+
+console.log(fullname1);
+console.log(fullname2);
+
+
+
+
+
+
+
+
+//Formal fullname
+function getFullname(firstname, surname, useFormalName = false) {
+    if (firstname === "" || surname === "") {
+      return "Invalid input, both firstname and surname are required.";
+    }
+    if (useFormalName) {
+      return "Lord " + firstname + " " + surname;
+    } else {
+      return firstname + " " + surname;
+    }
+  }
+
+var fullname1 = getFullname("Benjamin", "Hughes", true);
+var fullname2 = getFullname("Atefeh", "Behroozi");
+
+console.log(fullname1); 
+console.log(fullname2); 
+
+
+
+
+
+
+
+
+//Event application
+
+function getEventWeekday(days) {
+    const weekdays = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+    const today = new Date().getDay();
+    const eventDay = (today + days) % 7;
+    return weekdays[eventDay];
+   
+  }
+  console.log(getEventWeekday(5));
```

Comment:
each time we call the function we define the variable repeatedly in memory, for ex. weekdays . in this var. we are not doing any alteration inside a function, this is just a static variable with normal values. So better if you can keep this outside.

Reply:
Ok. Thanks

Reply:
I tried to change my codes and fixed my errors but I couldn't. I had problem with my Git but today I solved it, but now I don't know because of my Git or another thing! I have a question, I fixed my codes ones, 5 days ago, Now I want to know can I change it again or not?